modules/Rubrics/rubrics_edit_editRowsColumns.php

```diff
@@ -187,25 +187,29 @@
 						while ($rubricColumn = $result->fetch()) {
 							$row = $form->addRow();
 							$row->addLabel('columnName'.$count, sprintf(__('Column %1$s Title'), ($count + 1)));
+							$column = $row->addColumn()->addClass('right');
 
 							// Handle non-grade scale columns as a text field, otherwise a dropdown
 							if ($values['gibbonScaleID'] == '') {
-								$row->addTextField('columnTitle['.$count.']')
+								$column->addTextField('columnTitle['.$count.']')
 									->setID('columnTitle'.$count)
 									->maxLength(20)
 									->isRequired()
 									->setValue($rubricColumn['title']);
 							} else {
 								$data = array('gibbonScaleID' => $values['gibbonScaleID']);
 								$sql = "SELECT gibbonScaleGradeID as value, CONCAT(value, ' - ', descriptor) as name FROM gibbonScaleGrade WHERE gibbonScaleID=:gibbonScaleID AND NOT value='Incomplete' ORDER BY sequenceNumber";
-								$row->addSelect('gibbonScaleGradeID['.$count.']')
+								$column->addSelect('gibbonScaleGradeID['.$count.']')
 									->setID('gibbonScaleGradeID'.$count)
 									->fromQuery($pdo, $sql, $data)
 									->isRequired()
 									->selected($rubricColumn['gibbonScaleGradeID']);
 							}
 							$form->addHiddenValue('gibbonRubricColumnID['.$count.']', $rubricColumn['gibbonRubricColumnID']);
 
+							//$row = $form->addRow();
+							$column->addLabel('columnVisualise'.$count, sprintf(__('Visualise?'), ($count + 1)));
```

Comment:
Hmmm ... maybe this field needs a description. I'm not quite sure what `Visualise?` will mean in this particular context.

Reply:
I think a description would be good, but I was not sure how to format it. It looks rather clunky. Any suggestions from the table OO guru?